Tests appear to be failing due to listen port clashes:
```
--- FAIL: TestPeerPoolSimulationSuite (1.06s)
    --- PASS: TestPeerPoolSimulationSuite/TestMailServerPeersDiscovery (1.03s)
    --- PASS: TestPeerPoolSimulationSuite/TestPeerPoolCacheEthV5 (0.02s)
    --- FAIL: TestPeerPoolSimulationSuite/TestSingleTopicDiscoveryWithFailoverEthV5 (0.00s)
        assertions.go:239: 
                          
	Error Trace:	peerpool_test.go:87
            
	            	
			suite.go:88
            
	Error:      	Received unexpected error:
            
	            	listen tcp 0.0.0.0:33741: bind: address already in use
        peerpool_test.go:179: Skipping due to being flaky
FAIL
FAIL	github.com/status-im/status-go/peers	1.158s
```
Maybe there's a leftover process on that host that holds it from some build. I'm not sure if that port is static or randomly selected.

--- a/Makefile
+++ b/Makefile
@@ -220,11 +220,7 @@ release:
 
 gomobile-install:
 	go get -u golang.org/x/mobile/cmd/gomobile
-ifdef NDK_GOMOBILE
-	gomobile init -ndk $(NDK_GOMOBILE)
-else
 	gomobile init
-endif
 
 release-install:
 	go get -u github.com/c4milo/github-release

--- a/_assets/ci/Jenkinsfile.android
+++ b/_assets/ci/Jenkinsfile.android
@@ -2,6 +2,8 @@ pipeline {
   agent { label 'linux' }
 
   options {
+    timestamps()
+    disableConcurrentBuilds()
     /* Go requires a certain directory structure */
     checkoutToSubdirectory('src/github.com/status-im/status-go')
     /* manage how many builds we keep */
@@ -19,8 +21,9 @@ pipeline {
     PATH             = "${env.PATH}:${env.GOPATH}/bin"
     ANDROID_HOME     = '/usr/lib/android-sdk'
     ANDROID_SDK_ROOT = '/usr/lib/android-sdk'
-    ANDROID_NDK      = '/usr/lib/android-ndk'
-    ANDROID_NDK_HOME = '/usr/lib/android-ndk'
+    /* gomobile requires a specific NDK version */
+    ANDROID_NDK      = "${env.NDK_GOMOBILE}"
+    ANDROID_NDK_HOME = "${env.NDK_GOMOBILE}"
   }
 
   stages {

--- a/_assets/ci/Jenkinsfile.ios
+++ b/_assets/ci/Jenkinsfile.ios
@@ -2,6 +2,8 @@ pipeline {
   agent { label 'macos' }
 
   options {
+    timestamps()
+    disableConcurrentBuilds()
     /* Go requires a certain directory structure */
     checkoutToSubdirectory('src/github.com/status-im/status-go')
     /* manage how many builds we keep */

--- a/_assets/ci/Jenkinsfile.linux
+++ b/_assets/ci/Jenkinsfile.linux
@@ -2,6 +2,8 @@ pipeline {
   agent { label 'linux' }
 
   options {
+    timestamps()
+    disableConcurrentBuilds()
     /* Go requires a certain directory structure */
     checkoutToSubdirectory('src/github.com/status-im/status-go')
     /* manage how many builds we keep */

